packages/volto/src/components/manage/Form/Form.jsx

```diff
@@ -730,6 +793,7 @@ class Form extends Component {
                 history={this.props.history}
                 location={this.props.location}
                 token={this.props.token}
+                errors={this.state.errors.blocks}
```

Comment:
We should pass the whole error state to the BlocksForm (or introduce a `blocksError` prop to the BlocksForm.

I haven't looked in detail, but maybe also move the computing of blocks error to the BlocksForm? 

The idea is that some blocks need to have access to the whole form error state. For example, the volto-metadata-block exposes Dexterity content fields such as Title, etc and makes them editable from a block. If there's an error for that field, the error exists in the Form component, but the block doesn't have access to that whole state form error variable.

Reply:
Then the content type field is the one who has to have the validator, right? What's the point in having the same validator in the proxied metadata field?

Also, if you want to make an extra one, in the validator you have access to the full `formData` to check the current values. 

The point of doing it this way (also in a nested way in blocks) is because it does work in the same way for content types, forms and blocks so we don't have to come up with fancy new way of doing things. The whole blocks use case was implemented, literaly, withing 10 minutes, passing down the `error` prop.

IMO this is better than nothing which is the current state.

Reply:
Let's take this situation: I want to expose the content item title field as editable field in a block. I can initialize a Field for that field, but I also want to be able to pass the errors prop to that field. The errors is computed in the Form, and not passed down to the BlocksForm. All I'm saying is we should pass the whole state.errors to the BlocksForm, rather then just state.errors.blocks

Reply:
Done `blocksErrors` and `errors` props are passed down now.